From: uazo <uazo@users.noreply.github.com>
Date: Sat, 21 Sep 2024 14:38:40 +0000
Subject: Enable ClickToCall

Reactivation of the "click-to-call" flag and activation by default.
The functionality is only active if sending Intents from the web is allowed

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 chrome/browser/flag-metadata.json      | 5 -----
 components/sharing_message/features.cc | 3 +++
 2 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/chrome/browser/flag-metadata.json b/chrome/browser/flag-metadata.json
--- a/chrome/browser/flag-metadata.json
+++ b/chrome/browser/flag-metadata.json
@@ -1512,11 +1512,6 @@
     "owners": [ "sygiet@google.com", "vasilii@chromium.org" ],
     "expiry_milestone": 133
   },
-  {
-    "name": "click-to-call",
-    "owners": [ "ellyjones@chromium.org" ],
-    "expiry_milestone": 115
-  },
   {
     "name": "clipboard-history-longpress",
     "owners": ["newcomer@google.com", "multipaste@google.com"],
diff --git a/components/sharing_message/features.cc b/components/sharing_message/features.cc
--- a/components/sharing_message/features.cc
+++ b/components/sharing_message/features.cc
@@ -7,6 +7,9 @@
 #include "build/build_config.h"
 
 BASE_FEATURE(kClickToCall, "ClickToCall", base::FEATURE_DISABLED_BY_DEFAULT);
+#if BUILDFLAG(IS_ANDROID)
+SET_CROMITE_FEATURE_ENABLED(kClickToCall);
+#endif
 
 BASE_FEATURE(kSharingMatchPulseInterval,
              "SharingMatchPulseInterval",
--
